### ISSUE TITLE
Rename Tabs in Conversation Window

### ISSUE THREAD

**Issue by Aaarkay1** [ChatGPTBox-dev/chatGPTBox#382]
Thanks for this great extension! It'd be so nice if we can rename tabs in the Conversation Window which are currently named by date and time of their creation. I hope you listen. Thanks in advance.

**Comment by josStorer** [comment]
Glad to hear that you like this extension. As of now, I am actively maintaining another project ([RWKV-Runner](https://github.com/josStorer/RWKV-Runner)), so the updates for this extension may be slower. I will add this feature, but unfortunately it may not be done quickly.

**Comment by Aaarkay1** [comment]
That's totally ok. Everyone has right to work on whatever they want. Thanks for taking note of my suggestion. Here are few more suggestion which are trivial but would make it a great extension:
1. Option of resize the prompt window (Ctrl+B), especially width, which sometimes comes over the main text, making difficult to read the main website text.
2. I'd really like to know, if or when, Bard will be added.
3. Option to change the position of chat modules in the prompt  box from the settings

**Comment by josStorer** [comment]
I think I'll add Bard in 10 days
To avoid occlusion, currently you can use the storage function, and then open an independent window to use
<img width="380" alt="image" src="https://github.com/josStorer/chatGPTBox/assets/13366013/b139e31e-aa1a-4d10-8d6b-6cf00100bf30">
<img width="794" alt="image" src="https://github.com/josStorer/chatGPTBox/assets/13366013/0ffb90b9-6953-4a8f-a15d-ed4833a5a855">

**Comment by Aaarkay1** [comment]
[BUG/ISSUE] The prompt box takes the response box to the bottom  of the text when it is done writing the response. I mean, this should not happen. It should not scroll automatically to the end when chatbot stops writing.

**Comment by josStorer** [comment]
At that time, I did it intentionally because there was no significant finished prompt. I think I will remove it in the next version.

**Comment by josStorer** [comment]
@Aaarkay1 I almost forgot this
![image](https://github.com/josStorer/chatGPTBox/assets/13366013/017c4f3f-30ae-4a8b-90ce-ea44b7d8cad8)

**Comment by Aaarkay1** [comment]
I think you may have mistaken my issue. Here's example gif. In this example, you can see after writing the complete answer, the scroll automatically goes to the end. But i don't want that to happen as users usually start reading answers even when GPT is hasn't done writing the answer yet. But when it scrolls to the end by itself after writing complete writing, 
![1](https://github.com/josStorer/chatGPTBox/assets/63100754/5ad3877e-bb6b-4067-82e8-4ccb19254a97)

I also tried 'Focus to input box after answering' but it did not do what I wanted it to as it looks totally a different function.

**Comment by Wladefant** [comment]
Yes or another suggestion would be to just, let it rename by itself like chatgpt does

**Comment by josStorer** [comment]
@Wladefant 
The approach of ChatGPT is to send an additional request to generate a title based on the conversation content, but this means that users will incur additional costs. I believe many users do not want this (only ChatGPT web provides titles for free).

**Comment by Wladefant** [comment]
Yes, please use ChatGPT on the web when it's available. If there's no access, then just don't use it. I suppose that's not too bad, is it?
